### ISSUE TITLE
Callback gets called before css update

### ISSUE THREAD

**Issue by fcoelho** [nathansmith/adapt#16]
The callback function is passed the width of the screen, which is useful, but doesn't reflect _directly_ in the width of the containers.

Due to a jquery pĺugin I'm using, I have to set the width of some images based on the size of ".grid_12", which isn't defined before the callback since the css file has not been added yet to the document. I tried setting the 'onload' property of css but it didn''t work. Is there a way to call the callback just after the css has been actually loaded? (without some hacky setTimeout)

**Comment by nathansmith** [comment]
@fcoelho …

Unfortunately, there isn't a reliable way to watch for `onload` of a stylesheet.

Could you possibly do without a jQuery plugin, by simply doing this with CSS?

```
img {
  max-width: 100%;
}
```

**Comment by fcoelho** [comment]
@nathansmith 

Unfortunately it didn't work. I'm using the [jQuery Cycle](http://jquery.malsup.com/cycle/) plugin but it deliberately changes the image sizes for the slideshow.

I've tried using jquery with the load event,

``` javascript
$(css).load(function() {
    callback && callback(i, width);
});
```

but it will work only in Opera. Chrome for example doesn't really seem to respond to this. As of now I have modified adapt.js to fill in a new `<style id="current-css"></style>` container using this in `change()` (yes, I'm deliberately using jquery):

``` javascript
$.get(url, function(contents) {
    $("#current-css").html(contents);
    callback && callback(i, width);
});
```

So far it has worked fine for me and I'm happy with it. I feel like I'm cheating, but it tied together nicely.
